keps/sig-network/2819-CNI-traffic-shaping-support/kep.yaml

```
@@ -0,0 +1,35 @@
+title: CNI traffic shaping support
+kep-number: 2819
+authors:
+    - "@Lion-Wei"
+    - "@m1093782566"
+owning-sig: sig-network
+participating-sigs:
+status: implemented
```

Would that mean that this is generally available?
```suggestion
status: implementable
```